cli/add_domain.sh

```diff
@@ -31,41 +31,51 @@ REAL_CNAME=$(dig +short -t CNAME "$DOMAIN")
 REAL_CNAME=${REAL_CNAME:-"None"} # Weird hack to get "None" printed instead of a blank line
 
 if [ "$NEEDED_CNAME" != "$REAL_CNAME" ]; then
-    echo "The domain $DOMAIN does not have the correct CNAME. Expected: $NEEDED_CNAME, Actual: $REAL_CNAME"   
-	exit 1 
+	echo "The domain $DOMAIN does not have the correct CNAME. Expected: $NEEDED_CNAME, Actual: $REAL_CNAME"
+	echo "Falling back to TXT record checking..."
+	NEST_DOMAIN_VERIFICATION=$(dig +short -t TXT _nest_domain_verification.$DOMAIN | tr -d '"')
+	NEST_DOMAIN_VERIFICATION=${NEST_DOMAIN_VERIFICATION:-"None"}
```

Comment:
ik this is quite a nit but what if there's a user named `None`
I think it might be clearer if there's a separate `if` to handle no TXT record being found, which would point the user to the [guides page](https://guides.hackclub.app/index.php/Subdomains_and_Custom_Domains).